pages/maintenance.tsx

```diff
@@ -0,0 +1,14 @@
+import Seo from 'components/Layout/Seo/Seo';
+
+const Maintenance = (): React.ReactElement => (
+  <>
+    <Seo title="Maintenance mode" />
+    <h1>Maintenance mode</h1>
```

Comment:
should probably closely mirror content here: https://design-system.service.gov.uk/patterns/service-unavailable-pages/

Reply:
Agree with @jhackett1!